spock/mcp/mcpacket_extensions.py

```diff
@@ -349,7 +348,7 @@ def decode_extra(packet, bbuff):
     def encode_extra(packet):
         bbuff = utils.BoundBuffer()
         if packet.data['nbt'] is None:
-            packet.data['nbt'] = nbt._TAG_End()
+            packet.data['nbt'] = nbt._TagEnd()
```

Comment:
not sure, but if we change the `nbt` module, we could as well rename this to TagEnd